tests/integration/api/test_xai.py

```diff
@@ -177,6 +177,11 @@ def test_predict_with_explain(
         # Tickets: 142087, 141639
         return
 
+    if "yolox" in recipe:
+        # The cropping of the padded saliency maps is not implemented for OV (Model API) yet,
+        # so the saliency maps for PyTorch and OV are different.
+        return
```

Comment:
Is there a reason why the skip associated with this comment only happens in `yolox`?
And is there a ticket for this?